app/Models/Bible/BibleFileset.php

```diff
@@ -149,6 +128,7 @@ public function connections()
 
     public function organization()
     {
+        // BWF: 12/10/20 unsure whether to remove asset_id here
```

Comment:
This seems to be necessary when linking fileset_id -> bible -> organization -> asset_id

Reply:
removed comment